Formula/video-compare.rb

```diff
@@ -0,0 +1,21 @@
+class VideoCompare < Formula
+  desc "Split screen video comparison tool using FFmpeg and SDL2"
+  homepage "https://github.com/pixop/video-compare"
+  url "https://github.com/pixop/video-compare/archive/refs/tags/20220412.tar.gz"
+  sha256 "425eafd0095e8043c240658618c14c8a4e92954c41cd1adf5a3b01d275a3581c"
+  license "GPL-2.0-only"
+
+  depends_on "ffmpeg"
+  depends_on "sdl2"
+  depends_on "sdl2_ttf"
+
+  def install
+    system "make"
+    bin.install "video-compare"
+  end
+
+  test do
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thanks for the review!

I don't really see any meaningful way to provide such a test, since the only functionality of this tool is to provide a GUI for visual side-by-side comparison of videos for humans.

    Usage: video-compare [OPTIONS]... FILE1 FILE2

    -h, --help
        show help
    -d, --high-dpi
        allow high DPI mode for e.g. displaying UHD content on Retina displays
    -w, --window-size
        override window size, specified as [width]x[height] (e.g. 800x600)
    -t, --time-shift
        shift the time stamps of the right video by a user-specified number of milliseconds

The formula cookbook says 

> [...] and (despite their widespread use) foo --version and foo --help are bad tests. **However, a bad test is better than no test at all.**

So would it okay for you to proceed with the "bad" test?